.env

```
@@ -0,0 +1,10 @@
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_YWNjZXB0ZWQtY2hvdy05NS5jbGVyay5hY2NvdW50cy5kZXYk
+CLERK_SECRET_KEY=sk_test_mpfZe0XiSoV5SZgUn8ZqCNT9pVTu4fYMTAdcpIH8X7
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLERK_SECRET_KEY key should go before the NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)